src/python/bot/testcase_manager.py

```diff
@@ -1001,8 +1001,17 @@ def get_command_line_for_application(file_to_run='',
   content_shell_app_names = [
       'content_shell', 'content_shell.exe', 'Content Shell'
   ]
+
+  # In some cases, APP_REVISION is not an integer. In these cases, we assume
+  # the revisions are recent enough that we should remove the command line.
+  # This primarily affects impact tasks.
+  try:
+    revision = int(environment.get_value('APP_REVISION', 0))
+  except ValueError:
+    revision = 0
+
   if (environment.get_value('APP_NAME') in content_shell_app_names and
-      environment.get_value('APP_REVISION', 0) >= 558998):
```

Comment:
If bumping MIN_REVISION on these jobs to this point is an option, I'd rather just remove the hacks. WDYT?

Reply:
That seems much better to me. 558998 was back in May 2018 and seems old enough that we don't care about bisects to back there anymore? @inferno-chromium WDYT?

Reply:
Yes it might be already upped, check jobs and remove if needed.

Reply:
Removed.